activation/nipost_test.go

```diff
@@ -513,6 +534,10 @@ func TestNIPostBuilder_ManyPoETs_AllFinished(t *testing.T) {
 		},
 	}
 
+	poetCfg := PoetConfig{
+		GracePeriod: 1 * time.Second,
+	}
```

Comment:
Why was it needed?

Reply:
In an earlier commit I was checking this value inside `BuildNiPoST` but now it's just used to build the context passed to the PoET client. I removed it from the tests where that client is mocked and it doesn't need to be set.